docusaurus/docs/user-docs/settings/review-workflows.md

```diff
@@ -5,18 +5,40 @@ description: Managing your review workflows and stages in Strapi.
 
 # Managing Review Workflows <EnterpriseBadge />
 
-The Review Workflows feature allows you to create and manage any desired review stages for your content, enabling your team to collaborate in the content creation flow from draft to publication. Before being available in the [Content Manager](/user-docs/content-manager/reviewing-content) and [Content-Types Builder](/user-docs/content-type-builder), review workflows should be configured from ![Settings icon](/img/assets/icons/settings.svg) *Settings > Global settings > Review Workflows*.
+The Review Workflows feature allows you to create and manage different review workflows for different content types. Each workflow can consist of any review stages for your content, enabling your team to collaborate in the content creation flow from draft to publication. Before being available in the [Content Manager](/user-docs/content-manager/reviewing-content) and [Content-Types Builder](/user-docs/content-type-builder), review workflows should be configured from ![Settings icon](/img/assets/icons/settings.svg) *Settings > Global settings > Review Workflows*.
+
+- should we explain a use case for multiple workflows?
```

Comment:
Probably it could be a good idea to explain a use case :) 

An example of when to use multiple workflows could be when managing content for a large organization with different departments or teams, each responsible for creating and publishing content for various purposes. 

Each department or team might have its own review process, requiring different stages.